### DISCUSSION TITLE
How to output cell group to abaqus inp file?

### DISCUSSION THREAD

**Discussion by crest621010** [neperfepx/neper#894] [Q&A] [answered]
I set a cell group by "-group mode"  in Tessellation Module, and I want to retain this defined cell groups and output it to Abaqus inp file. How can I do it?  Thanks a lot.

NEPER="neper --rcfile none"
$NEPER -T -n from_morpho \
         -domain "cube(1,1,1)" \
         -morpho "diameq:0.4*dirac(0.2)+0.6*dirac(0.1)" \
         -morphooptiini coo:packing,weight:radeq \
         -morphooptistop itermax=1 \
         -dim 3 \
         -group mode \
         -o gene_grou_13
$NEPER -M gene_grou_13.tess \
      -rcl 2.0 \
      -interface cohesive \
      -format msh,inp \
      -order 1 \
      -faset none \
      -o gene_grou_13

**Comment by rquey** [comment, marked as answer]
This discussion has been closed as obsolete. Please repost if necessary.
